samples/v1.5/Test/RtlTestCard.json

```diff
@@ -179,12 +181,14 @@
         {
           "type": "Image",
           "url": "https://adaptivecards.io/content/cats/1.png",
+          "altText": "cat",
```

Comment:
Tiny nit but any reason the first letter isn't capitalized here but it is for the other alt texts?

Reply:
No reason. Good catch. I'll fix it.